lib/ansible/modules/cloud/amazon/ec2_vpc_net.py

```diff
@@ -16,61 +16,61 @@
 module: ec2_vpc_net
 short_description: Configure AWS virtual private clouds
 description:
-    - Create or terminate AWS virtual private clouds.  This module has a dependency on python-boto.
+    - Create, modify, and terminate AWS virtual private clouds.
 version_added: "2.0"
 author: Jonathan Davila (@defionscode)
```

Comment:
This is missing your authorship @s-hertel - a wide-ranging rewrite deserves attribution